dipy/reconst/ivim.py

```diff
@@ -381,7 +391,7 @@ def estimate_f_D_star(self, params_f_D_star, data, S0, D):
                 warningMsg += " as initial guess for leastsq. Parameters are"
                 warningMsg += " returned only from the linear fit."
                 warnings.warn(warningMsg, UserWarning)
-                f, D_star = params_f_D
+                f, D_star = params_f_D_star
```

Comment:
What's this change? Was this just a bug that was hiding in there? Looks like it.

Reply:
@arokem  yes! It was just a minor bug in the code.. I found it while reading it.

Reply:
@arokem Thank you for this! I will create a new issue and PR with the test for the same 👍